src/chain_spec.rs

```diff
@@ -300,7 +309,7 @@ fn testnet_genesis(
 			transaction_byte_fee: 0,
 		}),
 		sudo: Some(SudoConfig {
-			key: root_key,
+			key: root_key.clone(),
```

Comment:
Why did this have to be changed?

Reply:
Because I reuse `root_key` below in `forum_sudo` config. Rust asked me to make this change :)